docs/store-operations/translations/categories.mdx

```diff
@@ -0,0 +1,224 @@
+# Translations for Categories (Beta)
+
+<Callout type="info">
+The Translations Admin GraphQL API is currently available on Catalyst storefronts only.
+</Callout>
+
+
+The categories translatable fields are:
+
+* Name
+* Description
+* Page Title
+* Meta Keywords
+* Meta Description
+* Search Keywords
+
+## Examples
+
+Below are examples of GraphQL queries and mutations for retrieving and managing translation settings for categories.
+
+### Query translations
+
+This query returns a list of all translations currently supported by the platform.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```json filename="Example mutation: Query a translation" showLineNumbers copy
+        GRAPHQL {{host}}/stores/{{store_hash}}/graphql
+        X-Auth-Token: {{token}}
+
+      query {
+          store {
+              translations(filters: {
+                  resourceType: CATEGORIES,
+                  channelId: "bc/store/channel/3",
+                  localeId: "bc/store/locale/en"
+              }) {
+                  edges {
+                      node {
+                            resourceId
+                              fields {
+                                  fieldName
+                                  original
+                                  translation
+                              }
+                          }
+                      cursor
+                  }
+              }
+          }
+      }
+    ```
+  </Tab>
+<Tab>
+```json filename="Example query: Query a translation" showLineNumbers copy
```

Comment:
<img width="1108" alt="Screenshot 2025-04-08 at 10 29 14" src="https://github.com/user-attachments/assets/3ce108ad-7c73-4433-aa23-4e6b7ab5fc7c" />

Reply:
Hi @bc-yehor-kravchenko GitHub doesn't display the code correctly. It will look as like the following:
![Screenshot 2025-04-08 at 8 50 52 AM](https://github.com/user-attachments/assets/3f477304-b743-41b3-95e8-c1fab8ea3139)
![Screenshot 2025-04-08 at 8 50 44 AM](https://github.com/user-attachments/assets/de28cf3b-6395-4f5c-bf9f-ea18329eb2f6)